docs/management/connectors/action-types/servicenow.asciidoc

```diff
@@ -18,6 +18,8 @@ URL::       ServiceNow instance URL.
 Username::  Username for HTTP Basic authentication.
 Password::  Password for HTTP Basic authentication.
 
+The ServiceNow user requires at minimum read/create/update access to the Incident table and read access to the sys_choice. If access is not provided to sys_choice, then the choices will not render.
```

Comment:
What is `sys_choice`? Is it a table?

Reply:
Yes, this is a table on the ServiceNow side https://docs.servicenow.com/bundle/paris-platform-administration/page/administer/localization/reference/r_ChoicesTable.html which is used to populate dropdown values for incident form